components/Dates/ShowDatesList/DateListRow.js

```diff
@@ -0,0 +1,41 @@
+import FindPlace from "@/components/Places/FindPlace";
+import dayjs from "dayjs";
+import { StyledDatesList } from "./styled";
+import handelClickDate from "./HandelClickDate";
+
+// function handelClickDate(index, setListEntry) {
+//   setListEntry(index);
+// }
+
+export default function DateListRow({ date, index, setListEntry }) {
+  var weekOfYear = require("dayjs/plugin/weekOfYear");
+  dayjs.extend(weekOfYear);
+
+  const currentDate = dayjs().format("YYYY-MM-DD");
+  const tomorrowDate = dayjs().add(1, "day").format("YYYY-MM-DD");
+  const thisDate = date.data.date;
+
+  const relativTime =
+    thisDate === currentDate
+      ? "Heute"
+      : thisDate === tomorrowDate
+      ? "Morgen"
+      : dayjs(thisDate).week() === dayjs().week()
+      ? "Diese Woche"
+      : thisDate;
+
+  return (
+    <StyledDatesList
+      $hover
```

Comment:
`$` is a very strange prefix for a prop 🤔

Reply:
I thought this is a good way to pass a boolean as a prop.